tests/TestCase.php

```diff
@@ -16,4 +16,7 @@ public function createApplication()
 		return $app;
 	}
 
+	protected $connectionsToTransact = [
```

Comment:
This was necessary for the DatabaseTransactions class to know which connection to perform DB transactions in for the tests.